task13.cpp

```diff
@@ -0,0 +1,62 @@
+#include <iostream>
+#include <math.h>
+
+using namespace std;
+
+bool isNumberPrime(unsigned number) {
+    if (number <= 1) {
+        return false;
+    }
+
+    for (int i = 2; i * i <= number; i++) {
```

Comment:
I think it would be faster if you declared a variable before the for-cycle that holds value such as sqrt(number)